### PR TITLE
docs(core): add docs to Property and Event

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: modules/angular2/src/core/metadata/directives.ts [hunk 1]
```diff
@@ -417,7 +417,10 @@ export class DirectiveMetadata extends InjectableMetadata {
   selector: string;
 
   /**
-   * Enumerates the set of properties that accept data binding for a directive.
+   * Enumerates the set of data-bound properties for a directive
+   *
+   * Data-bound properties are automatically updated by Angular during change
```
Comment (btford) [inline modules/angular2/src/core/metadata/directives.ts]: Consider active voice: "Angular automatically updates data-bound properties during change detection."

Also is there a link to another doc page that you could add WRT "change detection" ?

File: modules/angular2/src/core/metadata/directives.ts [hunk 2]
```diff
@@ -940,38 +923,95 @@ export class PipeMetadata extends InjectableMetadata {
 }
 
 /**
- * Declare a bound field.
+ * Declares a data-bound property.
  *
- * ## Example
+ * Data-bound properties are automatically updated by Angular during change
```
Comment (btford) [inline modules/angular2/src/core/metadata/directives.ts]: Same: prefer active voice here

File: modules/angular2/src/core/metadata/directives.ts [hunk 3]
```diff
@@ -980,9 +1020,12 @@ export class EventMetadata {
 }
 
 /**
- * Declare a host property binding.
+ * Declares a host binding property.
  *
- * ## Example
+ * Data-bound properties are automatically updated by Angular during change
```
Comment (btford) [inline modules/angular2/src/core/metadata/directives.ts]: Same

**Comment by btford** [comment]
There are a few spots where you can profitably use active voice over passive voice.

Other than that LGTM

File: modules/angular2/src/core/metadata/directives.ts (hunk 1 above)
Comment (vsavkin) [inline modules/angular2/src/core/metadata/directives.ts]: I don't think we have a good guide on change detection. I will add a todo.

File: modules/angular2/src/core/metadata/directives.ts (hunk 2 above)
Comment (vsavkin) [inline modules/angular2/src/core/metadata/directives.ts]: Fixed

File: modules/angular2/src/core/metadata/directives.ts (hunk 3 above)
Comment (vsavkin) [inline modules/angular2/src/core/metadata/directives.ts]: Fixed

**Comment by vsavkin** [comment]
Merged via https://github.com/angular/angular/commit/ce6b364dc5a1e60030382559c0d59b5b51532a8d

**Pull request closed without merging**